tina/collectionsSchema/navigationBar.tsx

```diff
@@ -0,0 +1,134 @@
+const modals = ['BookDemo', 'EmailForm'];
+const iconOptions = ['FaCalendarDay', 'MdEmail'];
+
+export const navigationBarCollection = {
+  label: 'Navigation Bar',
+  name: 'navigationBar',
+  path: 'content/navigationBar',
+  format: 'json',
+  fields: [
+    {
+      name: 'navigationBar',
+      label: 'Navigation Bar',
+      type: 'string',
+    },
+    {
+      name: 'navItem',
+      label: 'Navigation Item',
+      list: true,
+      type: 'object',
+      templates: [
+        {
+          name: 'stringItem',
+          label: 'String Item',
+          ui: {
+            itemProps: (item) => {
+              return { label: '📄' + item?.label };
+            },
+          },
+          fields: [
+            {
+              name: 'label',
+              label: 'Label',
+              type: 'string',
+            },
+            {
+              name: 'href',
+              label: 'href',
+              type: 'string',
+            },
+          ],
+        },
+        {
+          name: 'groupOfStringItems',
+          label: 'Group Of String Items',
+          list: true,
+          type: 'object',
+          ui: {
+            itemProps: (item) => {
+              return { label: '📕' + item?.label };
+            },
+          },
+          fields: [
+            {
+              name: 'label',
+              label: 'Label',
+              type: 'string',
+            },
+            {
+              name: 'items',
+              label: 'Page or Submenu',
+              type: 'object',
+              list: true,
+              ui: {
+                itemProps: (item) => {
+                  return { label: item?.label };
+                },
+              },
+              fields: [
+                {
+                  name: 'label',
+                  label: 'Label',
+                  type: 'string',
+                },
+                {
+                  name: 'href',
+                  label: 'href',
+                  type: 'string',
+                },
+              ],
+            },
+          ],
+        },
+        //Note we are duplicating the ModalButton template from /components/blocks/ModalButton.template.ts but this is
+        //because this file is generated before hand and i still want to keep the ModalButton coupled into the components dir too
+
+        {
+          label: 'Modal Button',
```

Comment:
As per my conversation with @isaaclombardssw we agree this should not be duplicated.
We would rather reuse the existing code and keep it nice.

You should be able to import the template here.

e.g.   

https://github.com/SSWConsulting/SSW.Website/blob/35687156a480c0a809e61c12489a487cbae231c2/tina/collections/events.tsx#L22-L32

Reply:
✅ Done - i moved the fields of the `modalButton` out of its original schema and then imported just the fields into this template for the navbar collection as i just wanted the fields (and then added another field too) :)